### PR TITLE
Update item order

### PR DESCRIPTION
**Pull request by dspencer001**

### CODE REVIEW

File: client/js/_author/components/assessments/question_types/_question.jsx [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+      reorderActive: false,
+    };
+  }
+
+  getClassName() {
+    if (this.props.isActive && this.props.reorderActive) {
+      return 'reorder-active';
+    }
+
+    if (this.props.isActive) return 'is-active';
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/_question.jsx]: turnary?

File: client/js/_author/components/assessments/question_types/_question.jsx [hunk 2]
```diff
@@ -26,25 +67,29 @@ export default class Question extends React.Component {
...
+      <div
+        className={`o-item c-question ${className}`}
+        tabIndex="0"
+        onClick={() => this.props.activateItem(item.id)}
+        onFocus={() => this.props.activateItem(item.id)}
+      >
+        <QuestionHeader
           name={displayName.text}
-          type={genusTypeId}
+          type={genusTypeId}s
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/_question.jsx]: Whats the s

Comment (dspencer001) [inline client/js/_author/components/assessments/question_types/_question.jsx]: a typo

File: client/js/_author/components/assessments/question_types/question_common/inactive_h.jsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React    from 'react';
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/question_common/inactive_h.jsx]: Why single letter for filename

File: client/js/_author/components/assessments/question_types/question_common/reorder_h.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React    from 'react';
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/question_common/reorder_h.jsx]: whys single letter in file name

**Comment by BenHSmith13** [comment]
lgtm

**Pull request merged**